### ISSUE TITLE
Autoskill new command for death skip

### ISSUE THREAD

**Issue by sleeping-player** [MathewSachin/Fate-Grand-Automata#438]
#437 
I've been watching this issuetracker here for few weeks, and I've seen several features being rejected.  Among these, the recently requested function is skip of designated death.
Of course, you explained death detection impossible several times.
But those who write scripts know when death from Arash or Chen Gong will happen. Therefore, there is no need to "detect" what I already know, I just need to tell the script.
What we need now is to add a new custom command to the autoskill maker that will notify the designated death.
I will understand if this should be rejected for other reasons.

**Comment by MathewSachin** [comment]
I've rejected this too on Gamepress 😆 
Even if there's an AutoSkill command that says someone's going to die, there's no way to know after exactly how many seconds. NP times can vary depending on the servant and the phones performance, data loading for the first time.
If we don't know when the NP ended. So, how long will we keep on clicking?
If we click more than required, we waste time which is counter-intuitive to this feature.
If we click less, it's useless.
Since there's no way to detect death, there's no way to detect how much to click either.
Does that make sense?

Then there was a suggestion to keep clicking till we reach next wave, but there's no guarantee there will be a 'next' wave. It might be the last turn.

Then maybe we can check for both result screens and next wave, but we might skip even the result screen if we click too much.
So, we have to check for possibly all result screens or maybe just the materials screen 🤔 .
If something like a connection failure happens between clicking that way, we won't be able to retry.

I did this similar: keep clicking till material drop screen shows up for 'Screenshot Drops' and it failed miserably in a lot of cases, so I rewrote it a bit to change click locations.
But, maybe I'm just overthinking and we might be able to do this somehow 🤔.
